src/components/CampaignVisualisations/index.js

```diff
@@ -63,19 +63,34 @@ export default ({ visualisations }) => {
 };
 
 export const CrowdFundingVisualistation = ({ startnextId, goal, ...props }) => {
-  const [crowdFunding] = useGetCrowdfundingDirectly(startnextId);
+  // Check props includes a project key
```

Comment:
curios: why don't you also destructure  `project` here? to have nicer `in props` check? I kidna got used to seeing what props the function is gonna take the in the first line because... you know we don't have typescript ;-)

Reply:
The reason is because we need to check if the props even includes the `projects` property. If we destructure there, then we will either get the value or undefined. But we need to check if the props passed in even includes the project key, which we can't do if we destructure there.